lib/datasource/go/__snapshots__/index.spec.ts.snap

```diff
@@ -329,6 +329,7 @@ Array [
 
 exports[`datasource/go/index getReleases support bitbucket tags 1`] = `
 Object {
+  "registryUrl": "https://bitbucket.org",
```

Comment:
Why this change happened?

Reply:
I had to add this line so that no snapshots of the datasource changes. https://github.com/renovatebot/renovate/pull/10957/files#diff-74ff599c343f69422bac127d96420da26aeb58f0274040daedb002a1c81146f8R58

This change affects the Go datasource too as it uses bitbucket-tags. In this case only the default registry is returned explicitly now.